mppm_versioned_docs/version-1.0.0/dedicated-server/multiplayer-role.md

```diff
@@ -0,0 +1,23 @@
+---
+id: multiplayer-role
+title: Assign virtual players to a multiplayer role
+description: Learn how to use the dedicated server package to assign virtual players to a multiplayer role.
+---
+
+Install the [Dedicated Server package](https://docs.unity3d.com/Packages/com.unity.dedicated-server@1.0/manual/index.html) with Multiplayer Play Mode to switch players between the server and client role without a script.
```

Comment:
I would add an important requirement here : the dedicated server package has to have the same version as MPPM

Reply:
Thank you! Added a note.